Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs

```diff
@@ -864,6 +864,57 @@ public void OnAfterDeserialize()
             m_ScrollWheelAction = OnAfterSerializeActionProperty(m_ScrollWheelActionReference, m_ScrollWheelActionData);
         }
 
+        [SerializeField, HideInInspector] private InputActionAsset m_ActionsAsset;
+        internal InputActionAsset actionsAssetNoEnable
```

Comment:
This seems a bit unnatural as a property. A simple method with just the setter code seems more appropriate.

Reply:
Makes sense.